frontend/public/models/yaml-templates.ts

```diff
@@ -693,4 +693,48 @@ spec:
     type: JenkinsPipeline
   triggers:
   - type: ConfigChange
+`).setIn([referenceForModel(k8sModels.ResourceQuotaModel), 'rq-compute'], `
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: compute-quota
+  namespace: test-ns
+spec:
+  hard:
+    pods: '4'
+    requests.cpu: '1'
+    requests.memory: 1Gi
+    limits.cpu: '2'
+    limits.memory: 2Gi
+    requests.nvidia.com/gpu: 4
+`).setIn([referenceForModel(k8sModels.ResourceQuotaModel), 'rq-storageclass'], `
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: storage-quota
+  namespace: test-ns
+spec:
+  hard:
+    requests.storage: 100Gi
+    persistentvolumeclaims: 100
+    gold.storage-class.kubernetes.io/requests.storage: 3Gi
```

Comment:
Probably worth a comment in the YAML explaining that these must match a storage class name.